packages/hooks/src/useCookieState/__tests__/index.test.tsx

```diff
@@ -109,4 +110,49 @@ describe('useCookieState', () => {
     });
     expect(hook.result.current.state).toBe('hello world, zhangsan');
   });
+
+  it('using the same cookie name', () => {
+    const COOKIE_NAME = 'test-same-cookie-name';
+    const { result: result1 } = setUp(COOKIE_NAME, { defaultValue: 'A' });
+    const { result: result2 } = setUp(COOKIE_NAME, { defaultValue: 'B' });
+    expect(result1.current.state).toBe('A');
+    expect(result2.current.state).toBe('A');
+    act(() => {
+      result1.current.setState('B');
+    });
+    expect(result1.current.state).toBe('B');
+    expect(result2.current.state).toBe('A');
+    expect(Cookies.get(COOKIE_NAME)).toBe('B');
+    act(() => {
+      result2.current.setState('C');
+    });
+    expect(result1.current.state).toBe('B');
+    expect(result2.current.state).toBe('C');
+    expect(Cookies.get(COOKIE_NAME)).toBe('C');
+  });
+
+  it('delete cookie when cookie value equal undefined', () => {
+    const COOKIE = 'test-delete-undefined-cookie';
```

Comment:
`delete cookie when cookie value equal undefined` seems to be a little duplicated as `should support undefined`, Is it better to put them in one place?

Reply:
in addition, I pushed some commits, don't forget to pull before coding :)

Reply:
ok